MaxText/layers/quantizations.py

```diff
@@ -47,6 +48,16 @@ def einsum(self):
       )
     )
     return aqt_einsum
+  
+@dataclass
```

Comment:
This is great! Maybe we could have some class that looks like as the parent for the two? More of a nit
```
@dataclass
class Quantization:
    """Base class for quantization configurations"""

    def dot_general_cls(self):
        """ Placeholder for dot_general implementation in subclasses. """
        pass 
```

Reply:
Cool, Ill do that right away

Reply:
Done, added what you suggested.